src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/SeveraTest.kt

```diff
@@ -0,0 +1,58 @@
+package fi.metatavu.timebank.api.test.functional.tests
+
+import fi.metatavu.timebank.api.severa.SeveraAccessTokenContainer
+import fi.metatavu.timebank.api.severa.SeveraService
+import fi.metatavu.timebank.api.severa.SeveraService.Companion.USERS_READ
+import fi.metatavu.timebank.api.test.functional.resources.LocalTestProfile
+import fi.metatavu.timebank.api.test.functional.resources.TestWiremockResource
+import io.quarkus.test.common.QuarkusTestResource
+import io.quarkus.test.junit.QuarkusTest
+import io.quarkus.test.junit.TestProfile
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import javax.inject.Inject
+
+@QuarkusTest
+@QuarkusTestResource.List(
+    QuarkusTestResource(TestWiremockResource::class)
+)
+@TestProfile(LocalTestProfile::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class SeveraTest: AbstractTest() {
```

Comment:
This project doesn't have unit tests but rather integration or end-to-end tests. Meaning we test our own REST APIs and not the actual methods used within the software.

For example a test case could be that we:
- have a user in Keycloak with minimal information (for example no first or last names)
- have a matching user in our Severa mock
- verify that the missing information is correctly retrieved from Severa and persisted in Keycloak